lib/component-repository/src/routes/virtual-components/UpdateComponentConfig.js

```diff
@@ -1,3 +1,33 @@
+const ComponentConfig = require('../../models/ComponentConfig');
+const mongoose = require('mongoose');
 module.exports = async function (req, res, next) {
-    return next();
+  const { authClientId } = req.body;
+  const { _id } = req.virtualComponent;
+  const virtualComponentId = _id;
+  if (!mongoose.Types.ObjectId.isValid(virtualComponentId)) {
+    return res
+      .status(400)
+      .send({ errors: [{ message: 'Invalid id', code: 400 }] });
+  }
+
+  const qry = { virtualComponentId: virtualComponentId };
```

Comment:
The query should include the tenant also

Reply:
The tenant from the body or the tenant from the req.user.tenant ?

Reply:
I would say, if the req.user isAdmin then from the body, if not from the req.user

Reply:
Done